harpy/simulate_linkedreads.py

```
@@ -90,6 +92,7 @@ def linkedreads(genome_hap1, genome_hap2, output_dir, outer_distance, mutation_r
         "molecule_length" : molecule_length,
         "partitions" : partitions,
         "molecules_per_partition" : molecules_per,
+        "combine_haplotypes": combine_haplotypes,
```

Ruff (0.8.2)

Undefined name `combine_haplotypes`

(F821)